openex-front/src/admin/components/assets/endpoints/Endpoints.tsx

```diff
@@ -274,13 +296,19 @@ const Endpoints = () => {
                     className={classes.bodyItem}
                     style={inlineStyles.asset_tags}
                   >
-                    <ItemTags variant="list" tags={endpoint.asset_tags} />
+                    <ItemTags variant="list" tags={endpoint.asset_tags}/>
+                  </div>
+                  <div
+                    className={classes.bodyItem}
+                    style={inlineStyles.asset_status}
+                  >
+                    {differenceInHours(new Date().toISOString(), endpoint.asset_last_seen) > 6 ? t('Active') : t('Inactive')}
```

Comment:
why 6? maybe we can use a constant with a readable name instead of a magic number (?)

Reply:
Was decided for the first try.